src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -31,7 +34,33 @@ const en = ({ isDivorce, userCase, partner }: CommonContent) => ({
   applicantNotYetAppliedForConditionalOrder: `You can now apply for a ‘conditional order’.
   A conditional order is a document that says the court does not see any reason why you cannot
   ${isDivorce ? 'get a divorce' : 'end your civil partnership'}`,
+  conditionalOrderPending: {
+    beforeDueDate: {
+      line1: `You have applied for a conditional order. Your ${partner} also needs to apply
+      because this is a joint application ${isDivorce ? 'for divorce' : 'to end your civil partnership'}.
+      They have been sent an email to remind them.`,
+      line2: `If they do not apply by ${userCase.dueDate || dayjs().format('D MMMM YYYY')}
```

Comment:
this is not your pr but im kinda on the fence about using dayjs when the field doesnt exist because it gives false information.

Reply:
I'll change that